test/gurka/test_gtfs.cc

```diff
@@ -136,36 +146,21 @@ TEST(GtfsExample, WriteGtfs) {
     .location_type = gtfs::StopLocationType::StopOrPlatform, .wheelchair_boarding = "1",
   };
   feed.add_stop(first_stop_platform);
-  struct gtfs::Stop first_stop_station {
-    .stop_id = stopOneID, .stop_name = gtfs::Text("POINT NEMO"), .coordinates_present = true,
-    .stop_lat = station_one_ll->second.second, .stop_lon = station_one_ll->second.first,
-    .parent_station = "", .location_type = gtfs::StopLocationType::Station,
-    .wheelchair_boarding = "1",
-  };
-  feed.add_stop(first_stop_station);
 
-  struct gtfs::Stop second_stop_station {
+  struct gtfs::Stop second_stop_platform {
     .stop_id = stopTwoID, .stop_name = gtfs::Text("SECOND STOP"), .coordinates_present = true,
     .stop_lat = station_two_ll->second.second, .stop_lon = station_two_ll->second.first,
-    .parent_station = "", .location_type = gtfs::StopLocationType::Station,
+    .parent_station = "", .location_type = gtfs::StopLocationType::StopOrPlatform,
```

Comment:
i was hoping to cover all the cases in the test, a stop that has all 3 node types (egress/station/platform), a stop that has 2 of them (station/platform) and a stop that has just platform. this way we cover all permutations of generated not generated stuff